R/gcloud-install.R

```diff
@@ -24,17 +24,21 @@ gcloud_binary <- function() {
   if (!is.null(user_path))
     return(normalizePath(user_path))
 
-  candidates <- c(
-    function() Sys.which("gcloud"),
-    function() "~/google-cloud-sdk/bin/gcloud",
-    function() file.path(gcloud_binary_default(), "bin/gcloud")
-  )
-
   if (.Platform$OS.type == "windows") {
     appdata <- normalizePath(Sys.getenv("localappdata"), winslash = "/")
     win_path <- file.path(appdata, "Google/Cloud SDK/google-cloud-sdk/bin/gcloud.cmd")
```

Comment:
It looks like after this change `win_path` is no longer used (let's remove it if that's the case).

I'll merge anyway then we can make that fix after as this unblocks what might be a pretty common scenario.

Reply:
Thanks, good catch, I'll push the change to master.